src/main/java/com/minecolonies/coremod/entity/ai/util/ChatSpamFilter.java

```diff
@@ -1,6 +1,7 @@
 package com.minecolonies.coremod.entity.ai.util;
 
 import com.minecolonies.coremod.entity.EntityCitizen;
+import net.minecraft.util.text.TextComponentBase;
 import org.jetbrains.annotations.NotNull;
 
 import java.util.Arrays;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Arrays'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)